src/server/status_server/region_meta.rs

```diff
@@ -208,7 +207,6 @@ impl RegionMeta {
             raft_status: raft_group.status().into(),
             raft_apply: RaftApplyState {
                 applied_index: apply_state.get_applied_index(),
-                last_commit_index: apply_state.get_last_commit_index(),
```

Comment:
Like I commented in the doc, I think we can set it to `min(commit_index, persisted_index)` to do the check.